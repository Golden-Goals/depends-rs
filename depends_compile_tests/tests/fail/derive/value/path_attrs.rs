use depends::derives::Value;

#[derive(Value)]
#[depends(is::unhashable)]
struct Foo {
    bar: Vec<usize>,
}

fn main() {}
