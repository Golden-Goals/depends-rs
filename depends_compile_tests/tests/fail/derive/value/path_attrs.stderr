error: Unexpected attribute ""is""
 --> tests/fail/derive/value/path_attrs.rs:4:11
  |
4 | #[depends(is::unhashable)]
  |           ^^
